docs/modules/operation/pages/quick-start/users.adoc

```diff
@@ -12,13 +12,13 @@ Access the {page-component-title} web application at `\http://<ip-or-fqdn-of-you
 
 The default user login is `admin` with the password `admin`.
 
-The first time you log in, we prompt for permission to allow the `Data Choices` module to collect and publish anonymous usage statistics to https://stats.opennms.org.
+The first time you log in, we prompt for permission to allow the Data Choices module to collect and publish anonymous usage statistics to https://stats.opennms.org.
 The OpenNMS Group uses this anonymized information to help determine product usage and to improve the {page-component-title} software.
 We do not share it with third parties.
 
 image::users/data-sources.png[prompt,300]
 
-Click *Show me what is being sent* to see what xref:deep-dive/admin/housekeeping/introduction.adoc#ga-data-collection[information we collect].
+Click *Show me What is Being Sent* to see what xref:deep-dive/admin/housekeeping/introduction.adoc#ga-data-collection[information we collect].
```

Comment:
```suggestion
Click *Show me what is being sent* to see what xref:deep-dive/admin/housekeeping/introduction.adoc#ga-data-collection[information we collect].
```
We don't need to reproduce the exact capitalization on screen, especially when it's wrong. :-)

Reply:
Sounds good. I do seem to remember something about using title case for buttons in the Style Guide, but I can't find it right now. :p